src/main/java/io/stargate/sgv2/jsonapi/api/v1/NamespaceResource.java

```diff
@@ -41,11 +49,12 @@
 public class NamespaceResource {
```

Comment:
Can't add 2 resource for the same path, so added the createTable command to NamespaceResource. Not sure if this needs to be renamed to something else.

Reply:
Interesting, don't we want to have some prefix to distinguish the path
Like
"/v1/collection/{namespace}"
"/v1/table/{namespace}"

Just thinking loud.

But it can also explain that KeyspaceCommand and NamepsaceCommand are all SchemaCommand, so they use the same Resouce.
If we are gonna use one Resource, perhaps not call it NamespaceResource anymore.

Reply:
Changing path for existing Collection creation would be major incompatibility, fwtw. I guess we could add new one, deprecate old, but that would end up us supporting 2 paths practically forever.
So it may be best to add new path segment for "Table" case and leave "Collection" one the way it is, if possible.

Reply:
As per @amorton , the path needs to be same as collections. As for as name of the class he wants it to be left as is until further discussion done around it.